backend/src/models/mentor.ts

```diff
@@ -35,6 +36,7 @@ interface MentorDoc extends mongoose.Document {
   mentorMotivation: string;
   menteeIDs: mongoose.Types.ObjectId[];
   status: string;
+  personalAccessToken: string;
```

Comment:
Great! We also need to add the personalAccessToken to the signup page for now. 

Use this PR as reference cauz that adds 2 fields to signup flow - 
https://github.com/TritonSE/ALUM-Mobile-Application/pull/114